docs/custom-metric-filter.md

```diff
@@ -147,20 +152,168 @@ in Config Sync.
            exporters: [googlecloud/kubernetes]
          ```
 
-1. Apply the config map
+4. Apply the config map
```

Comment:
Why did the indexing change?

Reply:
They were from Intellij MD auto save, it converted them in-place. Fixed.